packages/hub/util/bin-utils.js

```diff
@@ -6,6 +6,7 @@ const fs = require('fs');
 
 const seedsFolder = 'seeds';
 const dataSourcesFolder = 'data-sources';
+const defaultTestingToken = 'testing-token';
```

Comment:
Please randomize this, I don't want it to be possible for people to accidentally hard code it anywhere.

It can be made available to the browser via a code generator. It can probably just go in plugin-utils/code-generators/environment.js

Reply:
it is in a code generator. I added it to the test-support package as that's the closest place to where it is consumed.

but yeah, I'll randomize it

Reply:
@ef4 so one aspect about this i feel a little uneasy about is that we only have a CI session token if the hub is started with that as an env var when in production mode. the CI session token allows unimpeded access. I feel a little uncomfortable allowing the ability to have unimpeded access tot he hub in production mode when you did specifically set that up, that's why we're using an env var to drive this in production mode. In the test env, using a random value for this is totally fine.